.github/workflows/test.yaml

```diff
@@ -0,0 +1,80 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+
+name: Test
+
+on:
+  push:
+    branches:
+      - 3.x
+      - master
+
+  pull_request:
+
+jobs:
+  test:
+    name: PHP ${{ matrix.php-version }} + ${{ matrix.dependencies }} + ${{ matrix.variant }}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+        dependencies: [highest]
+        variant: [normal]
+        include:
+          - php-version: "7.4"
+            dependencies: lowest
+            variant: normal
+          - php-version: "7.4"
+            dependencies: highest
+            variant: 'sonata-project/block-bundle:"dev-master as 3.x-dev"'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install PHP with extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          coverage: pcov
+          tools: composer:v2
+
+      - name: Add PHPUnit matcher
+        run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+      - name: Set Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache Composer
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}
```

Comment:
No need to take the `variant` in account?

Reply:
would be nice to add it, yes, will try